src/main/scala/li/cil/oc/client/gui/widget/Widget.scala

```diff
@@ -1,5 +1,8 @@
 package li.cil.oc.client.gui.widget
 
+import com.mojang.blaze3d.matrix.MatrixStack
+
+@Deprecated
```

Comment:
Vanilla minecraft now has a `net.minecraft.client.gui.widget.Widget` which is supported by all GUIs. Because there's only 1 widget class I'd like to part that over to avoid maintaining this.

Reply:
Oh, very nice, got it.